cscs-checks/apps/amber/amber_check.py

```diff
@@ -69,10 +69,10 @@ def __init__(self, **kwargs):
 
         self.reference = {
             'dom:mc': {
-                'perf': (5.0, -0.15, None)
+                'perf': (8.02, -0.05, None)
```

Comment:
I think that `8.02` is the same as `8.0`, so I would leave it as `8.0`. What do you think?